### ISSUE TITLE
Timers within Custom Events can not increment variables

### ISSUE THREAD

**Issue by Jeltr0n** [chrismaltby/gb-studio#1097]
**Describe the bug**
If I make a custom event that contains a timer, the timer works but incremented values within that timer do not work.

**To Reproduce**

1. Make a custom event with a timer
2. have the timer increment a variable
3. variable will remain the same.

**Expected behavior**
The variable will go up by 1 for each timer tick

**Screenshots**
![Custom Event timer bug](https://user-images.githubusercontent.com/70062490/163005686-c2af0b45-3c18-4c05-a473-3453368a92af.jpg)

**Platform (please complete the following information):**
3.0.3
Windows 10

**Comment by maxoakland** [comment]
I don’t believe it’s possible for more than one timer to run at a time. It’s more a thing to start in the Init of a scene

**Comment by Jeltr0n** [comment]
It is starting at the scene init and only one is running. It's just not incrementing the variable.
